public/app/features/dashboard/components/ShareModal/ShareLink.test.tsx

```diff
@@ -164,5 +176,20 @@ describe('ShareModal', () => {
         'http://server/#!/test?from=1000&to=2000&orgId=1&var-app=mupp&var-server=srv-01'
       );
     });
+
+    it('should shorten url', () => {
+      mockLocationHref('http://server/#!/test');
+      fillVariableValuesForUrlMock = (params: any) => {
+        params['var-app'] = 'mupp';
+        params['var-server'] = 'srv-01';
+      };
+      ctx.mount();
+      ctx.wrapper?.setState({ includeTemplateVars: true, useShortUrl: true }, async () => {
+        // @ts-ignore i dunno how else to await the async componentDidUpdate
+        await ctx.wrapper?.instance().busy;
```

Comment:
```suggestion
        await ctx.wrapper?.instance().buildUrl();
```
You can just await the `buildUrl`.